### DISCUSSION TITLE
Call script with activity manager and pass argument

### DISCUSSION THREAD

**Discussion by Android-X13** [termux/termux-app#2573] [Q&A] [answered]
I know from [here](https://github.com/termux/termux-app/issues/1399#issuecomment-569421617) how to successfully call termux scripts via `am`.

Is it possible to pass arguments to the script itself? I mean arguments that can be accessed by the script itself, with `$1` or `$@`.

**Comment by agnostic-apollo** [comment, marked as answer]
Use `RUN_COMMAND` intent. Linked way is not recommended anymore.

https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent

**Comment by Android-X13** [comment]
It works.
It appears that RUN_COMMAND_PATH does not accept variable names like $HOME, is that correct?

**Reply by agnostic-apollo** [reply]
For what? Use `~/` for executable and workdir.
